### ISSUE TITLE
Minor: Uncaught TypeError: activePane.getPath is not a function

### ISSUE THREAD

**Issue by tuomashatakka** [filipesabella/quick-file-actions#7]
### Reproducable by 

Creating a new file in any other view than in a text editor view*

- *Atom**: 1.22.0-beta1 x64
- **Electron**: 1.6.15
- **OS**: Microsoft Windows 10 Education
- **Thrown From**: [quick-file-actions](https://github.com/filipesabella/quick-file-actions) package 0.2.1

### Stack Trace

Uncaught TypeError: activePane.getPath is not a function

```
@ .atom\packages\quick-file-actions\lib\quick-file-actions.js:29

TypeError: activePane.getPath is not a function
    at /packages/quick-file-actions/lib/quick-file-actions.js:29:51)

    at CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.22.0-beta1/resources/app/src/command-registry.js:381:36)

    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (~/AppData/Local/atom/app-1.22.0-beta1/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:621:22)

    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (~/AppData/Local/atom/app-1.22.0-beta1/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:412:28)

    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (~/AppData/Local/atom/app-1.22.0-beta1/resources/app/src/window-event-handler.js:100:42)

    at HTMLDocument.<anonymous> (~/AppData/Local/atom/app-1.22.0-beta1/resources/app/src/window-event-handler.js:3:65)
```

### Commands

```
     -1:13.3.0 core:select-all (input.hidden-input)
     -1:11.2.0 core:confirm (input.hidden-input)
     -0:30.3.0 quick-file-actions:new (div.package-detail.panels-item)
     -0:23.8.0 intentions:highlight (input.hidden-input)
     -0:22.9.0 quick-file-actions:new (input.hidden-input)
     -0:18.2.0 core:confirm (input.hidden-input)
     -0:16.2.0 core:backspace (input.hidden-input)
     -0:12.1.0 intentions:highlight (input.hidden-input)
     -0:12 pane:show-next-recently-used-item (input.hidden-input)
     -0:11.7.0 pane:move-active-item-to-top-of-stack (input.hidden-input)
     -0:11 intentions:highlight (input.hidden-input)
     -0:10.9.0 pane:show-next-recently-used-item (input.hidden-input)
     -0:10.7.0 pane:move-active-item-to-top-of-stack (input.hidden-input)
     -0:10.5.0 intentions:highlight (input.hidden-input)
  2x -0:10.4.0 pane:show-next-recently-used-item (input.hidden-input)
     -0:09.1.0 pane:move-active-item-to-top-of-stack (div.package-detail.panels-item)
```

### Non-Core Packages

```
atom-clock 0.1.15 
atom-ternjs 0.18.3 
atom-typing-speedometer 0.3.1 
autocomplete-javascript 0.0.2 
busy-signal 1.4.3 
code-peek 1.4.21 
color-picker 2.2.5 
context-menu-plus 0.0.0 
declarations 1.1.3 
declarations-js 1.0.2 
docblockr 0.11.0 
eq-ui 0.0.0 
file-templates 0.0.0 
file-explorer 0.0.0 
file-hyperclick 0.1.0 
file-templates 0.0.0 
file-type-icons 1.3.4 
file-types 0.5.5 
flow-ide 1.8.1 
fold-functions 0.5.0 
git-split-diff-hyperclick 0.3.12 
github-notifications 0.1.2 
goto 1.8.3 
hyperclick 0.1.5 
hyperclick-markdown 0.2.2 
hyperclick-octolinker 0.1.3 
imdone-atom-harbour 2.0.18 
intentions 1.1.5 
intentions-colorpicker 1.0.1 
intentions-numbers 1.0.4 
js-hyperclick 1.11.1 
language-babel 2.74.2 
language-javascript-jsx 0.3.7 
language-jsx 0.1.1 
latex-completions 0.3.3 
linter 2.2.0 
linter-annotations 1.0.1 
linter-eslint 8.3.2 
linter-json-lint 0.1.2 
linter-ui-default 1.6.10 
linter-ui-hinted 0.0.5 
linter-xo 0.23.1 
marx 0.0.0 
path-hyperclick 0.3.0 
point-black-syntax 0.0.0 
power-bar 0.1.0 
quick-file-actions 0.2.1 
remember-file-positions 0.2.3 
split-diff 1.5.1 
stacktrace 0.0.2 
symbols-plus 0.1.1 
symbols-tree-nav 0.15.5 
toggler 0.3.0 
tool-bar-gui 0.1.2 
```

**Comment by filipesabella** [comment]
Ah good catch, I'll try to work on a fix.

Thanks for the report, @tuomashatakka.

**Comment by filipesabella** [comment]
Fixed in 0.2.2, again thanks for the report.
